### PR TITLE
feat(ux): open Web UI at launch on Linux

### PR DESCRIPTION
**Pull request by hacdias**
See https://github.com/ipfs-shipyard/ipfs-desktop/issues/1153#issuecomment-616243802.

Adds the option to open Web UI automatically on startup. This is enabled by default when the OS is neither Windows, nor macOS to help users that do not have a tray icon.

License: MIT
Signed-off-by: Henrique Dias <hacdias@gmail.com>

### CODE REVIEW

File: assets/locales/en.json [hunk 1]
```diff
@@ -218,6 +218,7 @@
     "openNodeSettings": "Open Node Settings",
     "appPreferences": "App Preferences",
     "launchOnStartup": "Launch at Login",
+    "launchWebUIOnStartup": "Launch Web UI on Startup",
```
Comment (jessicaschilling) [inline assets/locales/en.json:221]: Is it correct that this is different from “Launch at Login” (above) in that the former just starts Desktop in the background, but this launches Desktop and opens the Electron window as well? And that they’re mutually exclusive? If so, maybe the options should read (note order switches):
- Launch on Startup
- Background Launch on Startup

Note: This would mean re-translating the existing string in Transifex.

Comment (hacdias) [reply assets/locales/en.json:221]: This option means "open Electron window when IPFS Desktop starts" so they're not mutually exclusive

Comment (jessicaschilling) [reply assets/locales/en.json:221]: Sorry, I was wrong about "mutually exclusive" ... but is "launch with open Electron window on startup" vs "launch on startup but I don't need the Electron window, just the daemon and the menubar options" what we're asking here? If that's the case, I think the launch/background launch options still apply. Unless I'm just missing something really obvious! 😆

Comment (hacdias) [reply assets/locales/en.json:221]: I'm sorry, my English isn't the best some times. Let me try again to explain better, hopefully haha

- launchOnStartup --> Launch IPFS Desktop when the user logs in
- launchWebUIOnStartup -> Open the electron window (web ui) when IPFS desktop starts

For example, you can have the second one enabled without the first. Then, when you open IPFS Desktop, the web ui window opens too.

If you have both enabled, Desktop starts when the user logs in and the Web UI window pops up too.

If you have the first enabled and the second disabled, then IPFS Desktop starts when you login but you get no Window.

I hope that's clearer. And sorry again 😅

Comment (jessicaschilling) [reply assets/locales/en.json:221]: Ah, OK. I think I was missing the distinction between "system startup" and "IPFS Desktop startup". This makes more sense. Maybe then:

- "Launch at Login"
- "Open Web UI at Launch"

Using "launch" to bridge those two things together should clarify. Does that sound good to you and @lidel?

Side note: This is one of those situations where saying "Web UI" doesn't really make sense to anyone other than us. In a context like this one, I'd expect to see "Dashboard" or a similar term ... I know we've got other issues open to discuss this, though. Just pointing out an example 😊

Comment (hacdias) [reply assets/locales/en.json:221]: I agree with the Launch part and the Web UI part too. I agree we should not use Web UI on this context or on any UI at all. I like the 'Dashboard' idea.

Comment (jessicaschilling) [reply assets/locales/en.json:221]: I'd like to get @lidel's opinion on whether it makes sense to *start* saying "Dashboard" here when we use the phrase "Web UI" in various places already ... or whether we should have a separate effort to start deprecating the language "Web UI" globally. Both have pros and cons.

Comment (hacdias) [reply assets/locales/en.json:221]: Just leaving this here as a ref: https://github.com/ipfs-shipyard/ipfs-webui/issues/612

Comment (jessicaschilling) [reply assets/locales/en.json:221]: Based on that, probably worth leaving as "Web UI" until we have @olizilla back for more in-depth discussion.

Comment (hacdias) [reply assets/locales/en.json]: Just updated to "Open Web UI at Launch"

**Comment by lidel** [comment]
Rebased to be sure it works with the latest master and tests still pass.
